### DISCUSSION TITLE
Build for Android

### DISCUSSION THREAD

**Discussion by karlbrowns** [Haivision/srt#2018] [Q&A] [answered]
Is there anyone that has built srt-live-transmit for Android?  Is there a simple way of translating it to work with Android?

**Comment by maxsharabayko** [comment, marked as answer]
There is a dedicated [document](https://github.com/Haivision/srt/blob/master/docs/build/build-android.md). You might also try to ask your question in slack. [Join the conversation](https://slackin-srtalliance.azurewebsites.net/) in the `#development` channel on [Slack](https://srtalliance.slack.com).

**Reply by lgammo** [reply]
The 'document' assumes many things and it hardly sufficient to replicate the process. What is the environment? How is the environment setup? Where does this environment run? The 'document' and the scripts it mentions are all bash scripts. But I am running this on a Windows laptop. So do I run this inside cygwin (or a derivative like mobaxterm)? Can I do this from the AndroidStudio?
